src/data_info/routes/schemas/stats.clj

```diff
@@ -59,21 +71,49 @@
      :md5
      (describe String "The md5 hash of this file's contents, as calculated and saved by IRODS")}))
 
+(s/defschema FilteredStatInfo
+  (-> (merge DirStatInfo FileStatInfo)
+      (->optional-param :id)
+      (->optional-param :path)
+      (->optional-param :type)
+      (->optional-param :label)
+      (->optional-param :date-created)
+      (->optional-param :date-modified)
+      (->optional-param :permission)
+      (->optional-param :file-count)
+      (->optional-param :dir-count)
+      (->optional-param :file-size)
+      (->optional-param :content-type)
+      (->optional-param :infoType)
+      (->optional-param :md5)))
```

Comment:
ooh, that is pretty nice. I wonder if I can get a list of only optional keys out of the merged schemas so that I can remove another hardcoded list...